src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/ElectronicMonitoringProvider.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.ElectronicMonitoringProviderStatus
+
+@Schema(description = "Describes a electronic monitoring provider on a licence")
+data class ElectronicMonitoringProvider(
+
+  @Schema(description = "Internal identifier for this licence generated within this service", example = "123344")
+  val licenceId: Long,
```

Comment:
Guess we don't need to return this also as it'll will be retrieved in context of a licence